spec/services/import_occupation_standard_details_spec.rb

```diff
@@ -85,8 +85,8 @@
         ca = create(:state, abbreviation: "CA")
         ca_oa = create(:registration_agency, state: ca, agency_type: :oa)
 
-        create(:onet, code: "13-1071.01")
-        occupation = create(:occupation, rapids_code: "0157")
+        onet = create(:onet, code: "13-1071.01")
+        occupation = create(:occupation, onet: onet)
```

Comment:
@jenlu33 could you add an additional line in the spec file and check for a new record getting save? -- I want to keep this check for "0157" so that we have a case that accounts for rapid codes that start with a 0.

Reply:
@littleforest What would be the best way to do this? I noticed that `ImportOccupationStandardDetails` only takes in one row